File: 백준/Bronze/31617. 差 （Difference）/差 （Difference）.py
import sys
input=lambda:sys.stdin.readline().rstrip()
k=int(input())
n=int(input())
a=[*map(int,input().split())]
m=int(input())
b=[*map(int,input().split())]
cnt=0
for i in range(n):
    for j in range(m):
        if a[i]+k==b[j]:
            cnt+=1
print(cnt)

File: 백준/Bronze/31617. 差 （Difference）/差 （Difference）.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int k,n,m,cnt=0;
    cin>>k>>n;
    vector<int> a(n);
    for(int i=0;i<n;i++)
        cin>>a[i];
    cin>>m;
    vector<int> b(m);
    for(int i=0;i<m;i++)
        cin>>b[i];
    for(int i=0;i<n;i++) {
        for(int j=0;j<m;j++) {
            if(a[i]+k==b[j])
                cnt+=1;
        }
    }
    cout<<cnt;
    return 0;
}